src/components/organisms/news/news.html

```diff
@@ -0,0 +1,50 @@
+<div class="news">
+  <div class="news-and-events-title">
+    <h2>News<span class="caption">お知らせ</span></h2>
+  </div>
+  <div class="list">
+    <a href="#" class="text-decoration-none">
+      <div class="news-line">
+        <p class="news-event">2018.06.20</p>
+        <p class="news-text">
+          ブックカフェ・コワーキング系サイト向けテーマ「Swit...
```

Comment:
```suggestion
          ブックカフェ・コワーキング系サイト向けテーマ「Switch」をリリースしました。
```
ここは...で省略ではなくこうだと思います